### ISSUE TITLE
Option to customize links

### ISSUE THREAD

**Issue by maccman** [hanford/remark-slate#21]
This plugin is hardcoding the `link` property returned from parsing links.

https://github.com/hanford/remark-slate/blob/master/src/deserialize.ts#L99

In my Slate document I have the property as `url`. Would love a way to customize the property name.

**Comment by hanford** [comment]
Oh yeah that makes sense, we can add an option to customize the link destination key

**Comment by hanford** [comment]
Just published a new version, v1.4.0

You can customize this `key` by using `linkDestinationKey` (couldn't think of a better name 😅)

```
import remarkSlate from 'remark-slate'

unified()
  .use(parse)
  .use(remarkSlate, { linkDestinationKey: 'custom-value' })
  .process(value, function(err, file) {
    ...
  });
```
